content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -18,8 +18,8 @@ You can use _topology spread constraints_ to control how {{< glossary_tooltip te
 
 ### Enable Feature Gate
 
-Ensure the `EvenPodsSpread` feature gate is enabled (it is disabled by default
-in 1.16). See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/)
+Ensure the `EvenPodsSpread` feature gate is enabled (it is enabled by default
+since 1.18). See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/)
 for an explanation of enabling feature gates. The `EvenPodsSpread` feature gate must be enabled for the
 {{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}} **and**
 {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
```

Comment:
Once the feature gate goes beta, I'd be tempted to drop or simplify this section in a concept document. In a task document it might be more important to keep it.

For example, keep only this text:
```markdown
The `EvenPodsSpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/))
must be enabled for the
{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}} **and**
{{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
```